README.md

```diff
@@ -24,7 +24,8 @@ Unit testing of the application is performed within
 
 The `ccdb5_ui` (note the underscore) directory contains a thin [Django](https://www.djangoproject.com/)
 implementation that allows it to be used as a plugin for
-[CFPB's public website](https://github.com/cfpb/cfgov-refresh).
+[CFPB's public website](https://github.com/cfpb/cfgov-refresh). To run the Django
+tests, install [Tox](https://tox.readthedocs.io/en/latest/) and then run `tox`.
```

Comment:
Line 122-ish of this file also has python testing instructions. Do you want to update those to match these?

Reply:
@schbetsy great catch, will fix 👍